### ISSUE TITLE
API Manager deployment fails due to undefined url & protocol

### ISSUE THREAD

**Issue by abodalevsky** [Azure-Samples/AI-Gateway#24]
In my case, unfortunately with commented these 2 lines deployment has failed. 

https://github.com/Azure-Samples/AI-Gateway/blame/ff11430921b54d7452cad820ed6cf1c69318ed8a/labs/built-in-logging/main.bicep#L285

I believe, the same will be true for mock pool.
https://github.com/Azure-Samples/AI-Gateway/blame/ff11430921b54d7452cad820ed6cf1c69318ed8a/labs/built-in-logging/main.bicep#L302

**Comment by vieiraae** [comment]
Thanks Alexander! Are you using the latest az command line?

**Comment by abodalevsky** [comment]
Yes, the latest CLI available at that moment was used.  version: 2.60

**Comment by simonkurtz-MSFT** [comment]
Hi @abodalevsky, thank you for submitting this issue. My apologies it's taken so long. A larger PR #78 addresses this under the covers.
